### DISCUSSION TITLE
GetListingsItem

### DISCUSSION THREAD

**Discussion by ayotade** [abuzuhri/Amazon-SP-API-CSharp#508] [Q&A] [answered]
Hello,

I have just made my first call to get an amazon listing item and I got below:

[RateLimits ,ListingsItem_GetListingsItem]: 08/02/2023 12:23:45  Request/Burst: 1/10     Rate: 5.0/200ms
{
  "sku": "XXXXXXXXXX",
  "summaries": [
    {
      "conditionType": "new_new",
      "status": [
        "DISCOVERABLE"
      ],
      "marketplaceId": "XXXXXXXXXXXX",
      "asin": "XXXXXXXXXXX",
      "productType": "PET_FOOD",
      "itemName": "Eukanuba - Veterinary Remedy - Limited Calories - All Breeds Chicken - Dog - Pack of 1 (1 x 12000 g)",
      "createdDate": "2019-06-18T10:48:47.376Z",
      "lastUpdatedDate": "2022-12-16T09:51:47.329Z",
      "mainImage": {
        "link": "https://m.media-amazon.com/images/I/41z4gXdo5TL.jpg",
        "height": 500,
        "width": 500
      }
    }
  ]
}

There is nothing like the quantity and price of the item in this result. am I  missing something?

**Comment by ProNotion** [comment, marked as answer]
[See here](https://developer-docs.amazon.com/sp-api/docs/listings-items-api-v2021-08-01-reference) - the pricing is in the `ItemOffers`.

I've not tested it but you may want to try something like this:

```
  await _amazonConnection.ListingsItem.GetListingsItemAsync(new ParameterGetListingsItem
  {
      ...
      includedData = new List<Constants.ListingsIncludedData>
      {
          Constants.ListingsIncludedData.Offers
      }
  });
```

**Reply by ayotade** [reply]
Thank you for your help. Below is the error that I get: 

Unhandled Exception: FikaAmazonAPI.AmazonSpApiSDK.Models.Exceptions.AmazonNotFoundException: Resource that you are looking for is not found

**Reply by ayotade** [reply]
Thank you it is ok now. I put space in my SKUvalue by mistake.

**Reply by ayotade** [reply]
Please, how can I  get all the listed items on my amazon seller Central?. Kindly have a look at my code below and advise.

/* */var parameters = new ParameterReportList();
             parameters.pageSize = 100;
             parameters.reportTypes = new List<ReportTypes>();
             parameters.reportTypes.Add(ReportTypes.GET_MERCHANT_LISTINGS_DATA);
             parameters.marketplaceIds = new List<string>();
             parameters.marketplaceIds.Add(MarketPlace.UnitedKingdom.ID);
             var reports = amazonConnection.Reports.GetReports(parameters);
             foreach(var report in reports)
             {
                 Console.WriteLine(report.ToString());
             }
          
Thank you

**Reply by ProNotion** [reply]
[Take a look at the examples in the repository](https://github.com/abuzuhri/Amazon-SP-API-CSharp/blob/1229b5d5ad69951384b0f9be368b06eb0dec4f6f/Source/FikaAmazonAPI/ReportGeneration/ReportManager.cs#L190)

**Reply by ayotade** [reply]
I am extremely grateful. Thank you very much

**Reply by malafalx91** [reply]
I'm using this:

ReportManager reportManager = new ReportManager(amazonConnection);
                var products = reportManager.GetProducts(); //GET_MERCHANT_LISTINGS_ALL_DATA
